src/user/dto/qualification.dto.ts

```diff
@@ -1,30 +1,29 @@
-import { IsDefined, IsNotEmpty } from 'class-validator';
+import { IsDefined, IsNotEmpty } from "class-validator";
 
-import { Qualification, CareerProfile } from '@prisma/client';
+import { Qualification } from "@prisma/client";
 
-import { QualificationCreationDto } from './qualification-creation.dto';
+import { QualificationCreationDto } from "./qualification-creation.dto";
 
 export class QualificationDto extends QualificationCreationDto {
-  @IsNotEmpty()
-  id: string;
+    @IsNotEmpty()
+    id: string;
 
-  @IsDefined()
-  userCareerProfilId: string;
+    @IsDefined()
+    userCareerProfileId: string;
```

Comment:
`userCareerProfileId `is labeld with `@IsDefined()`, but not set via the constructor.